toolchain/install/busybox_info_test.cpp

```diff
@@ -294,7 +315,7 @@ TEST_F(BusyboxInfoTest, EnvBinaryPathOverride) {
   auto _ = llvm::make_scope_exit([] { unsetenv(Argv0OverrideEnv); });
```

Comment:
I guess you can drop this now (or better, test that `getenv(Argv0OverrideEnv) == nullptr` after calling `GetBusyboxInfo`)